mlrun/feature_store/feature_set.py

```diff
@@ -402,9 +417,12 @@ def save(self, tag="", versioned=False):
 
     def reload(self, update_spec=True):
         """reload/sync the feature vector status and spec from the DB"""
-        from_db = mlrun.get_run_db().get_feature_set(
+        from_db = self._get_run_db().get_feature_set(
```

Comment:
Now I understand where the `from_db` naming comes from 😂 , please fix here as well